pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,8 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            s = s.replace ("'", "%27").replace ("%", "%%");
+            cmd += (" \'" + PF.FileUtils.sanitize_path (s, null, false) + "\'");
```

Comment:
Would the modified sanitize method with the added `unescape` parameter be useful in the future for other places that need unescaped paths but not necessarily enclosing them in single quotes, hence not requiring the appinfo step of adding? I guess that could be done if the time comes / need arises.